### PR TITLE
Feature/host tests

### PR DESCRIPTION
**Pull request by TobiasMeissner2000**
#73 #74

### CODE REVIEW

File: Estimator/Estimator.Tests/Estimator.Tests.csproj [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
+    <PackageReference Include="Moq" Version="4.16.1" />
```
Comment (LeoEcker) [inline Estimator/Estimator.Tests/Estimator.Tests.csproj:9]: Bruchen wir das Mpq Pack noch?

File: Estimator/Estimator.Tests/Pages/CreateRoomTests.cs [hunk 2]
```diff
@@ -1,6 +1,60 @@
...
     public class CreateRoomTests
     {
+        [Theory]
+        [InlineData("Max Mustermann", false)]
+        [InlineData("", true)]
+        public void IsParameterEmptyTest(string userName, bool isUsernameEmpty)
+        {
+            #region Assign
+
+            var creatRoom = new CreateRoom();
```
Comment (TobiasMeissner2000) [inline Estimator/Estimator.Tests/Pages/CreateRoomTests.cs:15]: var creatRoom = new CreateRoom
{
     Username = userName;
};

File: Estimator/Estimator.Tests/Pages/CreateRoomTests.cs [hunk 3]
```diff
@@ -1,6 +1,60 @@
...
+
+            #region Act
+
+            var result = creatRoom.IsUsernameEmpty();
+
+            #endregion
+
+            #region Assert
+
+            Assert.Equal(isUsernameEmpty, result);
```
Comment (TobiasMeissner2000) [inline Estimator/Estimator.Tests/Pages/CreateRoomTests.cs:29]: UsernameIsEmpty

File: Estimator/Estimator.Tests/Pages/JoinRoomTests.cs [hunk 4]
```diff
@@ -1,6 +1,38 @@
...
+
+namespace Estimator.Tests.Pages
 {
     public class JoinRoomTests
     {
+        [Theory]
+        [InlineData("12345", "Max Mustermann", false)]
+        [InlineData("", "", true)]
+        [InlineData("123456", "", true)]
+        public void IsParameterEmptyTest(string roomId, string userName, bool isParameterEmpty)
```
Comment (TobiasMeissner2000) [inline Estimator/Estimator.Tests/Pages/JoinRoomTests.cs:14]: isParameterEmpty?

File: Estimator/Estimator/Pages/Host.razor [hunk 5]
```diff
@@ -117,14 +112,14 @@
                 </div>
 
                 <div class="row">
-                    @if (estimationClosed)
+                    @if (EstimationClosed)
```
Comment (LeoEcker) [inline Estimator/Estimator/Pages/Host.razor:115]: Warum groß?

File: Estimator/Estimator/Pages/CreateRoom.razor.cs [hunk 6]
```diff
@@ -25,19 +34,34 @@ private async void CreateNewRoom()
...
                 Trace.WriteLine(e);
-                await this.JsRuntime.InvokeVoidAsync("alert", "Something went wrong!\n Please try again.");
+                await this.Alert("Something went wrong!\n Please try again.");
             }
         }
 
-        private int ConvertType(string type)
+        internal bool IsUsernameEmpty()
+        {
+            return this.Username == String.Empty;
```
Comment (TobiasMeissner2000) [inline Estimator/Estimator/Pages/CreateRoom.razor.cs:49]: .Equals()

**Review by LeoEcker — APPROVED** [review]

File: Estimator/Estimator.Tests/Pages/CreateRoomTests.cs (hunk 2 above)
Comment (TobiasMeissner2000) [reply Estimator/Estimator.Tests/Pages/CreateRoomTests.cs:15]: done

File: Estimator/Estimator.Tests/Pages/CreateRoomTests.cs (hunk 3 above)
Comment (TobiasMeissner2000) [reply Estimator/Estimator.Tests/Pages/CreateRoomTests.cs:29]: done

**Pull request merged**